charts/dapr/charts/dapr_placement/templates/dapr_placement_deployment.yaml

```diff
@@ -6,7 +6,11 @@ metadata:
   labels:
     app: dapr-placement
 spec:
+{{- if eq .Values.global.ha.enabled true }}
```

Comment:
Please do not enable HA for placement in this iteration. It won't 100% work. Let's say we have 3 pods running for placement for HA mode and 10 applications are running.

3 apps can connect POD 1 of placement and the other can connect the other PODs of placement. in this case, all actors will be not discoverable. 

We're tracking this issue separately - https://github.com/dapr/dapr/issues/663

Reply:
Correct, placement should be set to 1 replica.